en/components/simple-combo.md

```diff
@@ -0,0 +1,108 @@
+
+<img class="responsive-img" src="../images/simple-combo_header.png" srcset="../images/simple-combo_header@2x.png 2x" />
+<img class="responsive-img" src="../images/simple-combo_item.png" srcset="../images/simple-combo_item@2x.png 2x" />
+
+## Dropdown Item States
+
+The Dropdown items within the Simple Combo support three different states: **Idle**, **Hover** and **Focused**. **Disabled** and **Selected** states can be switched on and off from a boolean property and they are applicable for all states of the Dropdown. The states of the Simple Combo Dropdown items are visually identical to the Combo Dropdown items without the checkbox icon for multiple selection.
+
+|  | Idle | Hover | Focused | Disabled |
+|  | ---- | ----- | ------- | -------- |
+|  | <img class="responsive-img" src="../images/simple-combo_item_idle.png" srcset="../images/simple-combo_item_idle@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_item_hover.png" srcset="../images/simple-combo_item_hover@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_item_focused.png" srcset="../images/simple-combo_item_focused@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_item_idle-disabled.png" srcset="../images/simple-combo_item_idle-disabled@2x.png 2x" /> |
+| Selected | <img class="responsive-img" src="../images/simple-combo_item_idle-selected.png" srcset="../images/simple-combo_item_idle-selected@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_item_hover-selected.png" srcset="../images/simple-combo_item_hover-selected@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_item_focused-selected.png" srcset="../images/simple-combo_item_focused-selected@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_item_selected-disabled.png" srcset="../images/simple-combo_item_selected-disabled@2x.png 2x" /> |
+
+## Styling
+
+The Simple Combo comes with the combined styling flexibility of its input and dropdown pieces. There are options for the input text elements' color and bottom line style and color. For the dropdown one may change the background color, or use the various options related to the items in the Simple Combo Dropdown, such as the search input and the items with their background and text colors.
+
+<img class="responsive-img" src="../images/simple-combo_styling.png" srcset="../images/simple-combo_styling@2x.png 2x" />
+
+## Usage
+
+When using the Simple Combo, its input and dropdown should have the same width and their left and right borders should match. Regarding how the dropdown is shown when the Simple Combo is focused, you should always show it on top of the content that follows the Simple Combo triggering its display. Combos don't push content like expansion panels do.
```

Comment:
Just a suggestion, but we could add an example, in which we show that you shouldn't use simple combo to select multiple items. It can be added on the Combo page as well (I already mentioned it there too).

Reply:
In my personal opinion, this is not needed.